internal/http/services/owncloud/ocdav/delete.go

```diff
@@ -76,7 +76,8 @@ func (s *svc) handleDelete(ctx context.Context, w http.ResponseWriter, r *http.R
 	case res.Status.Code == rpc.Code_CODE_OK:
 		return http.StatusNoContent, nil
 	case res.Status.Code == rpc.Code_CODE_NOT_FOUND:
-		return http.StatusNotFound, errtypes.NotFound("Resource not found") // mimic the oc10 error message
+		//lint:ignore ST1005 mimic the exact oc10 error message
+		return http.StatusNotFound, fmt.Errorf("Resource not found")
```

Comment:
If you aren't using string formats, you should use `errors.New("Resource not found")`.